README.md

```diff
@@ -31,14 +69,393 @@ Permissions to allow the user to interact with the support center:
+caseSeverity                  = "urgent"
+caseDesiredInstanceLimit      = 25
+caseStatusResolved            = "resolved"
+intervalAfterCaseCreationSecs = 30
+intervalBetweenChecksSecs     = 30
+
+// AccountPending indicates an account is pending
+AccountPending = "Pending"
+// AccountCreating indicates an account is being created
+AccountCreating = "Creating"
+// AccountFailed indicates account creation has failed
+AccountFailed = "Failed"
+// AccountReady indicates account creation is ready
+AccountReady = "Ready"
+// AccountPendingVerification indicates verification (of AWS limits and Enterprise Support) is pending
+AccountPendingVerification = "PendingVerification"
+// IAM Role name for IAM user creating resources in account
+accountOperatorIAMRole = "OrganizationAccountAccessRole"
+
+var awsAccountID string
+var desiredInstanceType = "m5.xlarge"
+var coveredRegions = []string{
+	"us-east-1",
+	"us-east-2",
+	"us-west-1",
+	"us-west-2",
+	"ca-central-1",
+	"eu-central-1",
+	"eu-west-1",
+	"eu-west-2",
+	"eu-west-3",
+	"ap-northeast-1",
+	"ap-northeast-2",
+	"ap-south-1",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"sa-east-1",
+}
+```
+
+### 3.2.3. Spec
+
+Updates the Account CR
+
+```
+spec:
+  awsAccountID: "000000112120"
+  claimLink: "claim-name"
+  iamUserSecret: accountName-secret
+```
+
+*awsAccountID* is updated with the account ID of the aws account that is created by the account controller
+*claimLink* holds the name of the accountClaim that has claimed this account CR
+*iamUserSecret* holds the name of the secret containing IAM user credentials for the AWS account
+
+### 3.2.4. Status
+
+Updates the Account CR
+```
+claimed: false
+conditions:
+- lastProbeTime: 2019-07-18T22:04:38Z
+  lastTransitioNTime: 2019-07-18T22:04:38Z
+  message: Attempting to create account
+  reason: Creating
+  status: "True"
+  type: Creating
+rotateCredentials: false
+state: Failed
+supportCaseID: "00000000"
+```
+
+*state* can be any of the account states defined in the constants
+*claimed* is true if `currentAcctInstance.Status.State == AccountReady && currentAcctInstance.Spec.ClaimLink != "`
+*rotateCredentials* when true rotates the temporary credentials on the next run
+*supportCaseID* is the ID of the aws support case to increase limits
+*conditions* indicates the last state the account had and supporting details
+
+### 3.2.5. Metrics
+
+Update in the account-controller
+```
+MetricTotalAWSAccounts
+```
+
+## 3.3. AccountClaim Controller
+
+The accountClaim-controller is triggered when an accountClaim is created in any namespace. It is responsible for following behaviours:
+
+1. Sets account `spec.ClaimLink` to the name of the accountClaim
+2. Sets accountClaim `spec.AccountLink` to the name of an unclaimed Account 
+3. Creates a secret in the accountClaim namespace that contains the credentials tied to the aws account in the accountCR
+4. Sets accountClaim `status.State = "Ready"` 
+
+### 3.3.1. Constants and Globals
```

Comment:
Again descriptions for these would be great

Reply:
@jharrington22 do you mind putting a suggestion with that these are used for I am not super familiar with the claim controller